submission_rules.adoc

```diff
@@ -269,7 +269,7 @@ MLCommons provides a web UI [TODO: URL] for verifying scores contained in the ta
 #### Post-submission grace period (submission deadline + 60 minutes):
 MLPerf will allow submissions for up to 60 minutes after the published deadline without explanation or penalty. The end of this grace period constitutes a hard deadline which will be strictly enforced.
 
-If a submitter cannot upload a valid submission tarball by the hard deadline due to some defect in the submission checker, a link to the tarball and a hash of the tarball must be sent to the chairs before the deadline. The burden is on the submitter to demonstrate that the submission could not be uploaded to via the web GUI because the checker was defective, and that the tarball passes the submission checker once the defect has been addressed.
+If a submitter cannot upload a valid submission tarball by the hard deadline due to some defect in the submission checker, a link to the tarball and a hash of the tarball must be sent to the submitters mailing list before the deadline. An issue must be raised on the relevant GitHub repository detailing the failure reason of the submission checker before the submission deadline. The burden is on the submitter to demonstrate that the submission could not be uploaded to via the web GUI because the checker was defective, and that the tarball passes the submission checker once the defect has been addressed. No other changes to the tarball other than those directly related to the submission checker fix must be allowed.
```

Comment:
```suggestion
If a submitter cannot upload a valid submission tarball by the hard deadline due to some defect in the submission checker, a link to the tarball and a hash of the tarball must be sent to the submitters mailing list before the deadline. An issue must be raised on the relevant GitHub repository detailing the failure reason of the submission checker before the submission deadline. The burden is on the submitter to demonstrate that the submission could not be uploaded to via the web GUI because the checker was defective, and that the tarball passes the submission checker once the defect has been addressed. No other changes to the tarball other than those directly related to the submission checker fix will be allowed.
```

Reply:
Thank you @DilipSequeira for the correction.